pkg/networkutils/network_test.go

```diff
@@ -224,25 +224,15 @@ func TestUpdateRuleListBySrc(t *testing.T) {
 			[]string{"10.10.0.0/16", "10.11.0.0/16"},
 			[]string{"10.12.0.0/16", "10.13.0.0/16"},
 			[]netlink.Rule{origRule},
-			make([]netlink.Rule, 4),
-			make([]*net.IPNet, 4),
-			[]int{origRule.Table, origRule.Table, origRule.Table, origRule.Table},
+			make([]netlink.Rule, 1),
```

Comment:
Nit: Some of these test parameters look like they're not useful anymore - and we can probably simplify these structures and the below code.

Reply:
ack. I will do it in a follow up commit.